### ISSUE TITLE
[ASUS GTX1080TI] Hash rate drops from 43 to 40 after DAG regeneration

### ISSUE THREAD

**Issue by ESP4Ever** [trexminer/T-Rex#1022]
Hello!

Thank you for such an amazing miner!
May I ask you to fix an issue with hash rate drops on ASUS GTX1080TI cards after T-Rex regenerates DAG?
It happens randomly, today it happened on epoch 470:

GPU #xxx: ASUS GTX 1080 Ti -  42.72 MH/s,              [T:55C, P:180W, F:27%, E:239kH/W], 1923/1923 R:0% I:0%
.....
20220129 12:58:05 GPU #xxx: generating DAG 4.67 GB for epoch 470 ...
......
GPU #xxx: ASUS GTX 1080 Ti -  40.43 MH/s,              [T:53C, P:176W, F:26%, E:270kH/W], 1923/1923 R:0% I:0%

The only way to recover now is to restart the miner.

Thank you in advance.

**Comment by sous-park** [comment]
i have the same poblem on my 1080ti zotac.
one year ago my crad had 43-44 Mh\s,  and the hash rate drops smoothly, already 41.5

**Comment by ESP4Ever** [comment]
It is a different story. That hash rate decrease can be compensated by overclock tuning with some power consumption increase.
In case of problem I described H/R drop recovers after the miner restart w/o additional tuning.

**Comment by sous-park** [comment]
> It is a different story. That hash rate decrease can be compensated by overclock tuning with some power consumption increase. In case of problem I described H/R drop recovers after the miner restart w/o additional tuning.

yes I know, but I have the problem with dropping hash rate after regeneration too and this has been going on for at least 6 months

**Comment by ESP4Ever** [comment]
H/R drop was expected due to DAG file growth
